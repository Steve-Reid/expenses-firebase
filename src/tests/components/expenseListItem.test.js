import React from 'react';
import { shallow } from 'enzyme';
import ExpenseListItem from '../../components/ExpensListItem';
import expenses from '../fixtures/expenses';

test('should render ExpenListItem with expense', () => {
  const wrapper = shallow(<ExpenseListItem {...expenses[1]} />);
  expect(wrapper).toMatchSnapshot();
});
