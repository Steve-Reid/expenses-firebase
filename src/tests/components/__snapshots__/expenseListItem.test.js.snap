// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenListItem with expense 1`] = `
<div>
  <Link
    replace={false}
    to="/edit/2"
  >
    <h3>
      Description: 
      Rent
    </h3>
  </Link>
  <p>
    Â£
    1,095.00
  </p>
  <p>
    28th Dec, 1969
  </p>
</div>
`;
